examples/animation/easing_functions.rs

```diff
@@ -37,6 +37,12 @@ fn setup(mut commands: Commands) {
         EaseFunction::QuinticIn,
         EaseFunction::QuinticOut,
         EaseFunction::QuinticInOut,
+        EaseFunction::SmoothStepIn,
+        EaseFunction::SmoothStepOut,
+        EaseFunction::SmoothStep,
+        EaseFunction::SmootherStepIn,
+        EaseFunction::SmootherStepOut,
+        EaseFunction::SmootherStep,
```

Comment:
adding those means that some of the functions are displayed outside of the window default size